frontend/packages/integration-tests-cypress/support/index.ts

```diff
@@ -40,9 +41,17 @@ after(() => {
 
 export const checkErrors = () =>
   cy.window().then((win) => {
-    if (win.windowError) {
-      throw new Error(`window/js runtime error detected: ${win.windowError}`);
-    }
+    const assertMsg =
+      win.windowError !== null
```

Comment:
Handle `undefined` as well.

```suggestion
      win.windowError
```

Reply:
fixed.